data_steward/cdr_cleaner/cleaning_rules/unit_normalization.py

```diff
@@ -0,0 +1,278 @@
+        rule of a class.  For example, if your class requires loading a static
+        table, that load operation should be defined here.  It SHOULD NOT BE
+        defined as part of get_query_specs().
+        :param client:
+        :return:
+        """
+
+        # creating _unit_mapping table
+        unit_mapping_table = f'{self.get_project_id()}.{self.get_dataset_id()}.{UNIT_MAPPING_TABLE}'
+        bq.create_tables(
+            client,
+            self.get_project_id(),
+            [unit_mapping_table],
+        )
+        # Uploading data to _unit_mapping table
+        unit_mappings_csv_path = os.path.join(resources.resource_files_path,
+                                              UNIT_MAPPING_FILE)
+        job = bq.upload_csv_data_to_bq_table(client, self.get_dataset_id(),
+                                             UNIT_MAPPING_TABLE,
+                                             unit_mappings_csv_path,
+                                             UNIT_MAPPING_TABLE_DISPOSITION)
+        try:
+            job.result()
+            LOGGER.info(
+                f"Created {self.get_dataset_id()}.{UNIT_MAPPING_TABLE} and loaded data from {unit_mappings_csv_path}"
+            )
+        except (BadRequest, OSError, AttributeError, TypeError, ValueError):
+            LOGGER.exception(
+                f"Unable to load data to {unit_mapping_table} table")
+
+    def get_query_specs(self):
+        """
+        :return:
+        """
+        sandbox_query = dict()
+        sandbox_query[cdr_consts.QUERY] = SANDBOX_UNITS_QUERY.render(
+            project_id=self.get_project_id(),
+            sandbox_dataset=self.get_sandbox_dataset_id(),
+            intermediary_table=self.get_sandbox_tablenames()[0],
+            dataset_id=self.get_dataset_id(),
+            unit_table_name=UNIT_MAPPING_TABLE,
+            measurement_table=MEASUREMENT)
+
+        update_query = dict()
+        update_query[cdr_consts.QUERY] = UNIT_NORMALIZATION_QUERY.render(
+            project_id=self.get_project_id(),
+            dataset_id=self.get_dataset_id(),
+            unit_table_name=UNIT_MAPPING_TABLE,
+            measurement_table=MEASUREMENT)
+        update_query[cdr_consts.DESTINATION_TABLE] = MEASUREMENT
+        update_query[cdr_consts.DISPOSITION] = bq_consts.WRITE_TRUNCATE
+        update_query[cdr_consts.DESTINATION_DATASET] = self.get_dataset_id()
+        return [sandbox_query, update_query]
+
+    def setup_validation(self, client):
+        """
+        Run required steps for validation setup
+        """
+        pass
+
+    def validate_rule(self, client):
+        """
+        Validates the cleaning rule which deletes or updates the data from the tables
+        """
+        pass
+
+    def get_sandbox_tablenames(self):
+        sandbox_table_names = list()
+        sandbox_table_names.append(self._issue_numbers[0].lower() + '_' +
+                                   self._affected_tables[0])
+        return sandbox_table_names
```

Comment:
This function `get_sandbox_tablenames()` might be more appropriate for rule documentation/logging than the one usage in `get_query_specs()` where it looks like we want a helper to fetch the name of the one sandbox table this rule happens to have. If this rule should need more than one sandbox table then `get_sandbox_tablenames()` won't be fit for this purpose (which list position is the one I'm looking for?). So I suggest a helper function like below. If this rule were eventually to evolve to need more than one sandbox table, then it could be refactored to accept a `base_name` parameter or something.

```suggestion
    def get_sandbox_table_name(self):
        return f'{self._issue_numbers[0].lower()}_measurement'

    def get_sandbox_tablenames(self):
        return [get_sandbox_table_name()]
```